products/apache-groovy.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Apache Groovy
+permalink: /apache-groovy
+alternate_urls:
+- /groovy
```

Comment:
one last thing might be adding /groovy-lang as alternate-urls

Reply:
(it is optional) so LGTM

Reply:
Done.